tests/handlers/test_freshmaker.py

```diff
@@ -77,6 +77,85 @@ def test_event_to_building(mock_koji_cs):
     assert build.state == 0
 
 
+@mock.patch('koji.ClientSession')
+def test_event_to_complete(mock_koji_cs):
+    """Test the Freshmaker handler when it receives an event to complete message."""
+    mock_koji_session = mock.Mock()
+    mock_koji_session.getTaskResult.side_effect = [
+        {
+            'koji_builds': ['710916']
+        },
+        {
+            'koji_builds': ['123456']
+        },
+        {
+            'koji_builds': ['234567']
+        }
+    ]
+    mock_koji_cs.return_value = mock_koji_session
+    event = FreshmakerEvent.get_or_create({
+        'id_': '2194',
+        'state': 1,
+        'state_name': 'BUILDING',
+        'url': 'http://freshmaker.domain.com/api/1/events/2194',
+        'event_type_id': 8,
+        'message_id':
+            'ID:messaging.domain.com-42045-1527890187852-9:1045742:0:0:1.RHBA-8018:0600-01'
+    })[0]
+    ContainerKojiBuild.get_or_create({
+        'id_': 710916,
+        'original_nvr': 'e2e-container-test-product-container-7.3-210.1523551880',
+        'state': 3
+    })[0].triggered_by_freshmaker_event.connect(event)
+    ContainerKojiBuild.get_or_create({
+        'id_': 123456,
+        'original_nvr': 'e2e-container-test-product-container-7.4-36',
+        'state': 3
+    })[0].triggered_by_freshmaker_event.connect(event)
+    ContainerKojiBuild.get_or_create({
+        'id_': 234567,
+        'original_nvr': 'e2e-container-test-product-container-7.5-133',
+        'state': 1
+    })[0].triggered_by_freshmaker_event.connect(event)
+    # Load the message to pass to the handler
+    with open(path.join(message_dir, 'freshmaker', 'event_to_complete.json'), 'r') as f:
+        msg = json.load(f)
+    # Make sure the handler can handle the message
```

Comment:
Optional: A new line here would be nice for logical separation of code. For instance the above code is like preparation code and the code below is actually executing the code to test.